app/views/shared/_booking_train_card.html.erb

```diff
@@ -0,0 +1,41 @@
+<div class="billboard-card">
+  <div class="card card-2">
+    <div class="card-content">
+      <div class="schedules">
+        <ul>
+          <li class="tab active">Inward</li>
+          <li class="tab">Outward</li>
+        </ul>
+        <div class="schedule">
+          <span>
+            <abbr><%= @booking.package.train_from.dep_city %> – <%=@booking.package.train_from.arr_city%></abbr>
+            <small>
+              <i><%= @booking.package.train_from.category %></i>
+              <p><%= @booking.package.start_date.strftime('%b %d, %Y') %>, <%= @booking.package.train_from.dep_time %> – <%= @booking.package.train_from.arr_time %></p>
+            </small>
+          </span>
+        </div>
+        <div class="schedule hidden">
+          <span>
+            <abbr><%= @booking.package.train_to.dep_city %> – <%=@booking.package.train_to.arr_city%></abbr>
+            <small>
+              <i><%= @booking.package.train_from.category %></i>
```

Comment:
should it be `train_to.category`?

Reply:
We will keep it for now. Let's discuss it later;)